.travis.yml

```diff
@@ -19,6 +19,8 @@ before_install:
   - curl -O https://capnproto.org/capnproto-c++-0.8.0.tar.gz && tar zxf capnproto-c++-0.8.0.tar.gz && cd capnproto-c++-0.8.0 && ./configure && make -j6 check && sudo make install && cd ../
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test && sudo apt-get -q update && sudo apt -y install g++-9 gcc-9 && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 90 --slave /usr/bin/g++ g++ /usr/bin/g++-9 --slave /usr/bin/gcov gcov /usr/bin/gcov-9
   - wd="$PWD" && cd .. && git clone https://github.com/catchorg/Catch2.git && cd Catch2 && cmake -Bbuild -H. -DBUILD_TESTING=OFF && sudo cmake --build build/ --target install && cd $wd
+  - mkdir ./tmp
+  - mkdir ./tmp/indices
```

Comment:
```suggestion
  - mkdir -p tmp/indices
```

Reply:
p stands for 'parent', meaning it recursively creates needed directories (it also keeps quiet if the dirs exist already)